# Elide call to face_for_font() when window system support is not included.
# See components/emacs/README.
#
--- emacs-24.3.orig/src/xdisp.c	2013-02-24 12:21:16.000000000 -0700
+++ emacs-24.3/src/xdisp.c	2013-09-24 10:45:08.854361844 -0600
@@ -6870,6 +6870,7 @@
 	}
     }
 
+#ifdef HAVE_WINDOW_SYSTEM
   /* Adjust face id for a multibyte character.  There are no multibyte
      character in unibyte text.  */
   if ((it->what == IT_CHARACTER || it->what == IT_COMPOSITION)
@@ -6910,6 +6911,7 @@
 	  it->face_id = FACE_FOR_CHAR (it->f, face, c, pos, it->string);
 	}
     }
+#endif
 
  done:
   /* Is this character the last one of a run of characters with
